man/systemd.network.xml

```diff
@@ -1130,6 +1130,39 @@
         </varlistentry>
       </variablelist>
   </refsect1>
+  <refsect1>
+    <title>[BridgeVLAN] Section Options</title>
+      <para>The <literal>[BridgeVLAN]</literal> section manages the
+      VLAN ID configuration of a bridge port and accepts the following
+      keys. Specify several <literal>[BridgeVLAN]</literal> sections to
+      configure several VLAN entries. The VLANFiltering= option has to be
+      enabled.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>VLAN=</varname></term>
+          <listitem>
+            <para>The VLAN ID allowed on the port. This can be either a
+            single ID or a range M-N.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>EgressUntagged=</varname></term>
+          <listitem>
+            <para>The VLAN ID specified here will be used to untag frames
+            on egress. This implies VLAN= above. This can be either a single
```

Comment:
What does "imply" mean here? If I specify only EgressUntagged= here, what does this set VLAN= to?

Also, please enclose VLAN= in `<varname>`